.github/workflows/build_and_deploy.yml

```
@@ -29,14 +29,13 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Get dependencies
-        run: |
-          go get -v -t -d ./src/...
+        run: go get -v -t -d
 
       - name: Build
-        run: GOARCH=amd64 GOOS=linux go build -o main ./src
+        run: GOARCH=amd64 GOOS=linux go build -o main
 
       - name: Test
-        run: go test -v ./src/
+        run: go test -v ./...
```

Have we checked if this can be:
```suggestion
        run: make test
```

---

Yes, done